File: Leetcode/1295/Linear_Search_Algo/main.cpp
class Solution {
    public:
        int findNumbers(vector<int>& nums) {
            int count = 0;
            for (int num: nums) {
                num = (int) (log10(num) + 1);
                if (num%2==0) count++;
            }
    
            return count;
            
        }
    };

File: Leetcode/1295/Linear_Search_Algo/main.kt
class Solution {
    fun findNumbers(nums: IntArray): Int {
        var count = 0
        for (num in nums) {
            val numm = (log10(abs(num).toDouble()) + 1).toInt()
            if (numm % 2 == 0) {
                count++
            }
        }
        return count
    }
}